static/docs/command-reference/run.md

```diff
@@ -152,9 +152,9 @@ hardware, etc.) and try to minize it (e.g. fix seeds).
   command is non-deterministic (meaning it produces different outputs from the
   same list of inputs).
 
-- `--remove-outs` - it removes stage outputs before executing the command. If
-  `--no-exec` specified outputs are removed anyway. This option is enabled by
-  default and deprecated. See `dvc remove` as well for more details.
+- `--remove-outs` - DEPRECATED - remove stage outputs before executing the
```

Comment:
I don't like using all upper case - can we for example adopt ~notation~? or (_deprecated_) should be enough

Reply:
OK, changed to (_deprecated_).